app/BUILD.bazel

```diff
@@ -214,16 +280,40 @@ in order to keep Gradle building.
 )
 for file in ALTERED_VIEW_MODELS]
 
+# Views
+
 # TODO(#1617): Remove genrules post-gradle
-# View files altered by genrules to be excluded in the app library.
+'''
+Source files for the views library that need to be altered by the view genrule.
+
+Place your file here if:
+ - It is a view file
+ - It imports resources (ex: `import org.oppia.app.R`)
```

Comment:
It's a bit confusing that both this & view models can depend on resources. How do we decide where non-views/view models that depend on resources go if they could go in either list?

Reply:
Wouldn't those files be kept in the app library if they could be? The only files that should be placed in a list like these are view files, listeners, annotation files, view models, and binding adapters, right?

Reply:
Okay, that makes sense and is clearer per your new comment at the top of the file & the revised per-list documentation.